client/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "slz-gui",
+  "version": "0.8.0",
+  "private": true,
+  "type": "module",
+  "repository": "git@github.com:carbon-design-system/carbon-tutorial.git",
+  "bugs": "https://github.com/carbon-design-system/carbon-tutorial/issues",
+  "license": "Apache-2.0",
+  "main": "/src/index.js",
+  "homepage": "https://pages.github.ibm.com/Alexander-Reiff/slzgui",
```

Comment:
needs to be updated

Reply:
Can you create the initial github page? I don't have access to the repo settings.